web/public/js/controllers/manage-dojo-users-controller.js

```diff
@@ -60,11 +62,243 @@ function cdManageDojoUsersCtrl($scope, $state, $q, cdDojoService, alertService,
+      class: 'cd-parent',
+      buttonText: $translate.instant('Change Role')
+    },
+    {
+      name: 'Volunteer',
+      userType: {
+        name:  'mentor'
+      },
+      approvalRequired: true,
+      image: userUtils.defaultAvatar('mentor'),
+      text: $translate.instant('Mentor/Volunteer - these volunteers power their local Dojos with their technical and organisational skills and inspire the next generation of coders, entrepreneurs and innovators!'),
+      class: 'cd-volunteer',
+      buttonText: $translate.instant('Change Role')
+    },
+    {
+      name: 'Champion',
+      userType: {
+        name:  'champion'
+      },
+      approvalRequired: true,
+      image: userUtils.defaultAvatar('champion'),
+      text: $translate.instant('Mentor/Volunteer - these volunteers power their local Dojos with their technical and organisational skills and inspire the next generation of coders, entrepreneurs and innovators!'),
+      class: 'cd-champion',
+      buttonText: $translate.instant('Change Role')
+    }
+  ];
+
+  $scope.youthRoles = [
+    {
+      name: 'Attendee',
+      image: userUtils.defaultAvatar('ninja'),
+      approvalRequired: false,
+      text: $translate.instant('Attendees at CoderDojo are young people under 18 who attend and make CoderDojo cool. Often known as Ninjas, CoderDojo attendees come to the Dojo to have fun and bring their awesome ideas to life.'),
+      class: 'cd-parent',
+      buttonText: $translate.instant('Change Role')
+    },
+    {
+      name: 'Youth Volunteer',
+      userType: {
+        name:  'mentor'
+      },
+      approvalRequired: true,
+      image: userUtils.defaultAvatar('mentor'),
+      text: $translate.instant('Mentor/Volunteer - these volunteers power their local Dojos with their technical and organisational skills and inspire the next generation of coders, entrepreneurs and innovators!'),
+      class: 'cd-volunteer',
+      buttonText: $translate.instant('Change Role')
+    },
+    {
+      name: 'Youth Champion',
+      userType: {
+        name:  'champion'
+      },
+      approvalRequired: true,
+      image: userUtils.defaultAvatar('champion'),
+      text: $translate.instant('Mentor/Volunteer - these volunteers power their local Dojos with their technical and organisational skills and inspire the next generation of coders, entrepreneurs and innovators!'),
+      class: 'cd-champion',
+      buttonText: $translate.instant('Change Role')
+    }
+  ];
+
+  $scope.awardBadgeAction = {
+    icon: 'trophy',
+    title: $translate.instant('Award Badge'),
+    ngShow: function () {
+      return $scope.selectedItems.length === 1;
+    },
+    ngInclude: '/dojos/template/badges/award-form',
+    showAsAction: 'always'
+  };
+
+  function updateActions() {
+    $scope.actions = [{
```

Comment:
Move to template? (We already have one moved (award badge) which is done purely because we use transclusion, so is necessary. We should be consistent)